modules/recommended-etcd-practices.adoc

```diff
@@ -25,7 +27,9 @@ The following hard disk features provide optimal etcd performance:
 * RAID 0 technology for increased performance.
 * Dedicated etcd drives. Do not place log files or other heavy workloads on etcd drives.
 
-Avoid NAS or SAN setups and spinning drives. Always benchmark by using utilities such as fio. Continuously monitor the cluster performance as it increases.
+Avoid NAS or SAN setups and spinning drives. Ceph RBD, as well as other network-attached storage, can result in unpredictable network latency. If possible, use PCI passthrough with NVM devices instead. 
```

Comment:
Spell out NAS, SAN, RBD, PCI, and NVM, on first occurrence. 

I do not think are well-known acronyms or abbreviations.

Reply:
As well as > and

![Screenshot from 2023-06-09 14-38-46](https://github.com/openshift/openshift-docs/assets/57954076/ed7f6b4e-8683-42ff-8681-23ab14b81061)

Reply:
> If possible, use PCI passthrough with NVM devices instead.

"Instead" seems to dangle or be misplaced with the priority it's trying to convey. How about:

> If possible, use PCI passthrough with NVM devices instead of other X.

I'm not sure what generic components are being compared.

Reply:
I also think that this paragraph reads as a note and should be enclosed in admonition syntax. WDYT?

Reply:
I'd beg to differ about most of those acronyms given that our audience consists of people who are looking to optimize etcd performance. In my judgment, though, happy to spell out RBD. 

```suggestion
[NOTE]
====
Avoid NAS or SAN setups and spinning drives. Ceph Rados Block Device (RBD), as well as other network-attached storage, can result in unpredictable network latency. To provide fast storage to etcd nodes at scale, use PCI passthrough to pass NVM devices directly to the nodes. 
====
```